_posts/2021-08-11-persistent-identifiers.md

```diff
@@ -0,0 +1,84 @@
+---
+title:  "Exposing and resolving persistent identifiers on the web"
+author: Rachel Heaven
+categories:
+  - Open Data
+tags:
+  - Linked Data
+  - RDF
+---
+
+# Exposing and resolving persistent identifiers on the web
```

Comment:
This heading can be removed, as the page title is generated by the `title:` field on line 2

Reply:
removed